### DISCUSSION TITLE
Scene Viewer failing while loading a 3d model

### DISCUSSION THREAD

**Discussion by hd-esteban** [google/model-viewer#2222] [Q&A]
Scene viewer is failing while loading this 3d model.

https://drive.google.com/file/d/1h_kIhH0Qxlgw0x-C8Q2iJY30SaIr8LTg/view?usp=sharing

We tried to verify the file using this tool: https://arvr.google.com/scene-viewer-preview but any error is reported, could you please advice on this?

![Screenshot_20200909_074111_com google android googlequicksearchbox](https://user-images.githubusercontent.com/1881501/92601962-183c7e80-f273-11ea-9ae9-3c47755e71b4.jpg)

### Browser Affected

- [X ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [X ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
@tpsiaki who can advise on the SceneViewer side.

**Comment by tpsiaki** [comment]
@Esteban26 the attached gltf uses absolute paths to resource files in the gltf.  Scene viewer only supports relative paths for resource uris.

**Comment by AIRS-EXR** [comment]
I meet the same problem. I cannot use https to show the model, but it works well on http. please tell me how to fix it?

**Comment by elalish** [comment]
@AIRS-EXR if it's really the same problem then the same solution should fix it. Can you give us a link to your site so we can see what's not working?

**Comment by Sheldonfrith** [comment]
@tpsiaki I think I'm having the same problem as the OP, just trying to figure out what you mean exactly with your reply. Are you saying model-viewer only works in scene viewer with model files hosted on the same server? Or am I misunderstanding what you mean by 'relative paths'? Thanks!
